examples/common.h

```diff
@@ -63,6 +63,16 @@ extern int lg2_tag(git_repository *repo, int argc, char **argv);
  */
 extern void check_lg2(int error, const char *message, const char *extra);
 
+/**
+ * Read a file path
+ *
+ * path: a file path (char *)
+ *
+ * returns char * with the contents of the file (must be freed) or NULL if
+ * file does not exist or memory allocation failed
```

Comment:
We use a custom documentation format that's similar to Javadoc. So this should probably be something like:

```
/**
 * Read a file into a buffer
 * @param path: path to the file that shall be read
 * @return NUL-terminated buffer if the file was successfully read, NULL-pointer otherwise
 */
```

Reply:
Done